### DISCUSSION TITLE
Add Penny DEL Hockey

### DISCUSSION THREAD

**Discussion by chrissyshw** [vasqued2/ha-teamtracker#183] [Ideas]
Hi, this integration is very nice and works well. I just have a feature request for the League section. Is it possible to add the german "Penny DEL" Hockey League to this integration? 

This would be so gread.

Thank you and best regards,
Christopher

**Comment by vasqued2** [comment]
Doesn't look like ESPN has an API for it.  If you find one, you should be able to configure a custom API.

[https://sports.core.api.espn.com/v2/sports/hockey/leagues?lang=en&region=us](https://sports.core.api.espn.com/v2/sports/hockey/leagues?lang=en&region=us)

```
  "items": [
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/hockey-world-cup?lang=en&region=us"
    },
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/mens-college-hockey?lang=en&region=us"
    },
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/nhl?lang=en&region=us"
    },
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/olympics-mens-ice-hockey?lang=en&region=us"
    },
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/olympics-womens-ice-hockey?lang=en&region=us"
    },
    {
      "$ref": "http://sports.core.api.espn.com/v2/sports/hockey/leagues/womens-college-hockey?lang=en&region=us"
    }
  ]
```
